SingularityUI/app/components/common/modal/ModalWrapper.es6

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 
-export const getClickComponent = (component) => (
+export const getClickComponent = (component, doFirst = (() => Promise.resolve())) => (
```

Comment:
i think you can replace `() => Promise.resolve()` with a method reference (`Promise.resolve`)